### PR TITLE
Added 2.9 release notes.

### PR DESCRIPTION
**Pull request by AWSHurneyt**
### Description
Added 2.9 release notes.

### Issues Resolved
https://github.com/opensearch-project/index-management-dashboards-plugin/issues/796

### Check List
- [x] Commits are signed per the DCO using --signoff

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.
For more information on following Developer Certificate of Origin and signing off your commits, please check [here](https://github.com/opensearch-project/OpenSearch/blob/main/CONTRIBUTING.md#developer-certificate-of-origin).

### CODE REVIEW

**Review by bowenlan-amzn — APPROVED** [review]

File: release-notes/opensearch-index-management-dashboards-plugin.release-notes-2.9.0.0.md [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+
+### Bug Fixes
+* Fix flush flaky test. ([#779](https://github.com/opensearch-project/index-management-dashboards-plugin/pull/779))
+* Fix CI on Node.js v18. ([#785](https://github.com/opensearch-project/index-management-dashboards-plugin/pull/785))
+* Fix: add validation on data.method when using transport.request. ([#801](https://github.com/opensearch-project/index-management-dashboards-plugin/pull/801))
+
+### Refactoring
+* Update release note and minor change to toast message. ([#770](https://github.com/opensearch-project/index-management-dashboards-plugin/pull/770))
+* Minor change for clear cache operation. ([#773](https://github.com/opensearch-project/index-management-dashboards-plugin/pull/773))
+* Wording changes. ([#784](https://github.com/opensearch-project/index-management-dashboards-plugin/pull/784))
```
Comment (Hailong-am) [inline release-notes/opensearch-index-management-dashboards-plugin.release-notes-2.9.0.0.md:20]: These three were included in 2.8, should we remove them from 2.9 release notes

File: release-notes/opensearch-index-management-dashboards-plugin.release-notes-2.9.0.0.md [hunk 2]
```diff
@@ -0,0 +1,23 @@
...
+### Maintenance
+* Increment version to 2.9.0.0. ([#795](https://github.com/opensearch-project/index-management-dashboards-plugin/pull/795))
+* Move index management from plugin section to management section. ([#741](https://github.com/opensearch-project/index-management-dashboards-plugin/pull/741))
+
+### Features
+* Implemented alias action UX. ([#754](https://github.com/opensearch-project/index-management-dashboards-plugin/pull/754))
+
+### Bug Fixes
+* Fix flush flaky test. ([#779](https://github.com/opensearch-project/index-management-dashboards-plugin/pull/779))
+* Fix CI on Node.js v18. ([#785](https://github.com/opensearch-project/index-management-dashboards-plugin/pull/785))
```
Comment (Hailong-am) [inline release-notes/opensearch-index-management-dashboards-plugin.release-notes-2.9.0.0.md:14]: same for 785 and 779, were included in 2.8 release notes

**Review by Hailong-am — CHANGES_REQUESTED** [review]

File: release-notes/opensearch-index-management-dashboards-plugin.release-notes-2.9.0.0.md (hunk 2 above)
Comment (AWSHurneyt) [reply release-notes/opensearch-index-management-dashboards-plugin.release-notes-2.9.0.0.md]: Thank you! Removed.

File: release-notes/opensearch-index-management-dashboards-plugin.release-notes-2.9.0.0.md (hunk 1 above)
Comment (AWSHurneyt) [reply release-notes/opensearch-index-management-dashboards-plugin.release-notes-2.9.0.0.md]: Thank you! Removed.

**Review by amsiglan — APPROVED** [review]

**Review by lezzago — APPROVED** [review]

**Review by getsaurabh02 — APPROVED** [review]

**Pull request merged**